src/setup-ganache.ts

```diff
@@ -22,10 +22,10 @@ export type AccountConfig = {
 
 export type NodeOptions = {
   accounts: AccountConfig[];
-  gasLimit: number;
-  gasPrice: string;
   coverage: boolean;
-  allowUnlimitedContractSize: boolean;
+  gasPrice?: string;
+  gasLimit?: number;
```

Comment:
Yes, because Ganache expects `gasPrice` to be hex string. These are options for Ganache not for users, they are not exposed to users. Users still expected to provide `gasPrice` as a number in `test-environment` config.